### ISSUE TITLE
Export Community (to ZIP)

### ISSUE THREAD

**Issue by heathergreerklein** [DSpace/DSpace#2842]
From release plan spreadsheet

Estimate from release plan: 8 hours
Expressing interest: none

Depends on "Scripts & Processes" endpoint

**Comment by abollini** [comment]
Do we need to implement a "SAF" option under the Export menu or a button on the community home page to use as shortcut? or both? if we need to add a button shortcut in the collection home should we have one (standard mode?) or two as in the previous versions to distinguish between the migrate and the standard mode?

**Comment by abollini** [comment]
@tdonohue should this go in 7.4? we confirm the estimation and we would like to claim the task.
This is similar but not the same than https://github.com/DSpace/dspace-angular/issues/774 and can be done together maybe saving some time

**Comment by tdonohue** [comment]
Yes, this can go into 7.4 alongside, as my assumption is it's mostly implemented in https://github.com/DSpace/DSpace/pull/8318 already (it might even be entirely implemented in that PR, assuming that PR can export from a Community UUID)

**Comment by tdonohue** [comment]
@abollini : Another older ticket previously assigned to your team.  Should this and https://github.com/DSpace/dspace-angular/issues/774 stay assigned to your team for 7.5?  If not, I'll unassign them.
